android/src/main/java/io/ably/flutter/plugin/AblyFlutterMessage.java

```diff
@@ -11,6 +13,13 @@
         this.handle = handle;
         this.message = message;
     }
+
+    @NonNull
+    @Override
+    public String toString() {
+        return "{message=" + (message.toString()) + ", handle=" + (handle==null?"":handle.toString()) + "}";
```

Comment:
Please include spaces for clarity. This is too compressed. This is better:

```suggestion
        return "{message=" + (message.toString()) + ", handle=" + (handle == null ? "" : handle.toString()) + "}";
```

Reply:
Same as https://github.com/ably/ably-flutter/pull/23#discussion_r471921091

052467d338e6fd8a37e3474c03b355e0ec1b8fdc